dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.h

```diff
@@ -126,7 +126,7 @@ class SegmentReadTaskPool
         auto blk_avg_bytes = total_count > 0 ? total_bytes / total_count : 0;
         auto approximate_max_pending_block_bytes = blk_avg_bytes * max_queue_size;
         auto total_rows = blk_stat.totalRows();
-        LOG_DEBUG(
+        LOG_INFO(
```

Comment:
1. Is there only 1 SegmentReadTaskPool for one TableScan?
2. If it helps locate the performance issue of ReadThreads, consider adding it to ScanContext?